core/modules/loader/CentralWorker.cc

```diff
@@ -0,0 +1,1146 @@
+// -*- LSST-C++ -*-
```

Comment:
This file is huge. I happen to review it many months ago. And it's not clear from tis PR what were the  changes to it in a context of this development.